### PR TITLE
[Mentor] -Add new mentors to add to the OTAP page.

### PR DESCRIPTION
**Pull request by Sopaphorn**
❗️❗️Warning❗️❗️
- Check new mentors name, information and picture of mentors everyone, that is correct or not ?
- Thai name is correct but English name I am not sure.

image

### CODE REVIEW

File: pages/mentors.vue [hunk 1]
```diff
@@ -131,7 +131,48 @@
               topics="Agile, Web, Mobile (Android), Chatbot"
               linkedin="ajahncharles"
             >
-              Winner LINEHACK. Co&#8209;founder Phits.Tech. Polyglot programmer.
+              ผู้ชนะ LINEHACK ผู้ร่วมก่อตั้ง Pits.Tech โปรแกรมเมอร์หลายภาษา
```
Comment (antonyharfield) [inline pages/mentors.vue:134]: You spelt Phits Tech wrong!

File: pages/mentors.vue [hunk 2]
```diff
@@ -131,7 +131,48 @@
...
+            >
+              บริษัท พาณิชย์ดิจิทัล จำกัด
+            </mentor>
+          </div>
+          <div class="column is-half-tablet is-one-third-desktop is-one-quarter-widescreen">
+            <mentor
+              :image-src="require('~/assets/images/people/Anuwut.jpg')"
+              name="Anuwat jongyindee"
+              thai-name="อนุวัฒน์ จงยินดี"
+              topics="Financial account, Strategy, Good governance risk management"
```
Comment (antonyharfield) [inline pages/mentors.vue:162]: Change to `Finance, Strategy, Risk management`

File: pages/mentors.vue [hunk 3]
```diff
@@ -131,7 +131,48 @@
...
+              name="Wanitcha Wankhawisun"
+              thai-name="วณิชชา วรรคาวิสันต์"
+              topics="Online marketing"
+            >
+              บริษัท พาณิชย์ดิจิทัล จำกัด
+            </mentor>
+          </div>
+          <div class="column is-half-tablet is-one-third-desktop is-one-quarter-widescreen">
+            <mentor
+              :image-src="require('~/assets/images/people/Anuwut.jpg')"
```
Comment (antonyharfield) [inline pages/mentors.vue:159]: Keep file names to be lowercase.

File: pages/mentors.vue [hunk 4]
```diff
@@ -131,7 +131,48 @@
...
+              name="Anuwat jongyindee"
```
Comment (antonyharfield) [inline pages/mentors.vue:160]: Names *ALWAYS* start with capital letter.

File: pages/mentors.vue [hunk 5]
```diff
@@ -131,7 +131,48 @@
...
+              บริษัทพฤกษาโฮลดิ้ง จำกัด (มหาชน)
+            </mentor>
+          </div>
+          <div class="column is-half-tablet is-one-third-desktop is-one-quarter-widescreen">
+            <mentor
+              :image-src="require('~/assets/images/people/workshop1-speaker3.jpg')"
+              name="Supot Kulprangthong"
+              thai-name="สุพจน์ กุลปรางค์ทอง"
+              topics="Strategy, Marketing"
+              linkedin="สุพจน์-กุลปรางค์ทอง-36080779/th"
```
Comment (antonyharfield) [inline pages/mentors.vue:173]: Check.

**Review by antonyharfield — CHANGES_REQUESTED** [review]

File: pages/mentors.vue (hunk 1 above)
Comment (Sopaphorn) [reply pages/mentors.vue:134]: I edited

File: pages/mentors.vue (hunk 2 above)
Comment (Sopaphorn) [reply pages/mentors.vue:162]: I changed.

File: pages/mentors.vue (hunk 3 above)
Comment (Sopaphorn) [reply pages/mentors.vue:159]: ok

File: pages/mentors.vue (hunk 4 above)
Comment (Sopaphorn) [reply pages/mentors.vue:160]: I am sorry I edited

**Review by antonyharfield — APPROVED** [review]

**Pull request merged**